Tests/DatadogTests/Datadog/DatadogCore/BatteryStatusReaderTests.swift

```diff
@@ -1,71 +0,0 @@
-/*
- * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
- * This product includes software developed at Datadog (https://www.datadoghq.com/).
- * Copyright 2019-Present Datadog, Inc.
- */
-
-import XCTest
-@testable import Datadog
-
-class BatteryStatusReaderTests: XCTestCase {
```

Comment:
Deleting this file - it wasn't used anywhere (even not referenced in Xcode project).